publishing does not work

the build fails with:

```
FAILURE: Build failed with an exception.

* Where:
Script '/home/marten/workspace/gitversion/gver-dsl/publish.gradle' line: 7

* What went wrong:
A problem occurred evaluating script.
> Could not set unknown property 'classifier' for task ':gver-dsl:javadocJar' of type org.gradle.api.tasks.bundling.Jar.
```

--- a/gver-dsl/publish.gradle
+++ b/gver-dsl/publish.gradle
@@ -4,12 +4,12 @@ if (project.hasProperty('SONATYPE_USERNAME') && project.hasProperty('SONATYPE_PA
     apply plugin: 'signing'
 
     task javadocJar(type: Jar, dependsOn: javadoc) {
-        classifier = 'javadoc'
+        archiveClassifier = 'javadoc'
         from javadoc.destinationDir
     }
 
     task sourceJar(type: Jar) {
-        classifier "sources"
+        archiveClassifier = "sources"
         from sourceSets.main.allJava
     }
 

